956-feature-flag-beans/src/main/java/io/harness/beans/FeatureName.java

```diff
@@ -419,6 +419,7 @@ public enum FeatureName {
   FFM_8344_FLAG_CLEANUP("Enable Stale Flag Cleanup in Feature Flags filtered listing page", HarnessTeam.CF),
   FFM_8823_AIDA_SDK_CODE_SAMPLE("Enable AIDA generated code sample in FF Onboarding", HarnessTeam.CF),
   FFM_8184_FEATURE_FLAG_TAGGING("Enable tagging in Feature Flags", HarnessTeam.CF),
+  FFM_9497_PROXY_KEY_MANAGEMENT("Enable proxy key management in Feature Flags", Harness.CF),  
```

Comment:
Looks like this needs to be HarnessTeam.CF here
```suggestion
  FFM_9497_PROXY_KEY_MANAGEMENT("Enable proxy key management in Feature Flags", HarnessTeam.CF),  
```

Reply:
ah brilliant find @conormurray95 thanks so much have updated!